src/sema/yul/expression.rs

```diff
@@ -386,6 +386,13 @@ pub(crate) fn resolve_function_call(
 
     if let Some(built_in) = parse_builtin_keyword(func_call.id.name.as_str()) {
         let prototype = &built_in.get_prototype_info();
+        if !prototype.is_available(&ns.target) {
+            ns.diagnostics.push(Diagnostic::error(
+                func_call.loc,
+                format!("builtin '{}' is not available for target {}. Please, open a GitHub issue if there is need to support this function.", prototype.name, ns.target)
```

Comment:
What do you think of including a link to our issue page?

Reply:
That sounds cools. I avoided the link to make the message shorter, but I can add it anyways.

Reply:
Due to this error, Solang will never reach the `unimplemented!` function in codegen.